### PR TITLE
(#5181) - add an issue template

### PR DESCRIPTION
**Pull request by nolanlawson**
I think this is worth trying out. I'm hoping it can result in higher-quality
issues, where we don't need to ask users to provide more details about their error.

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+### Adapter (IndexedDB/WebSQL/LevelDB/etc.)
+
+(Write here.)
+
+### Server (CouchDB/Cloudant/Couchbase/PouchDB Server/etc.)
+
+(Write here.)
+
+### Please describe your issue. Test cases are very helpful!
```
Comment (nickcolley) [inline ISSUE_TEMPLATE.md]: Maybe be more explicit what 'test case' means, not so sure it's obvious exactly what we're asking for here?

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<!--
+     Hello! 👋
+     Thank you for filing an issue on PouchDB.
```
Comment (nickcolley) [inline ISSUE_TEMPLATE.md]: super nit: filing -> opening?

**Comment by nickcolley** [comment]
This seems helpful to me! 👍

**Comment by nolanlawson** [comment]
I like your changes. I'll make those changes and then merge!

**Pull request merged**